cmd/kubeadm/app/preflight/checks.go

```diff
@@ -142,8 +142,8 @@ func (sc ServiceCheck) Check() (warnings, errorList []error) {
 
 	if !initSystem.ServiceIsEnabled(sc.Service) {
 		warnings = append(warnings,
-			errors.Errorf("%s service is not enabled, please run 'systemctl enable %s.service'",
-				sc.Service, sc.Service))
+			errors.Errorf("%s service is not enabled, please run '%s'",
+				sc.Service, initSystem.EnableCommand(sc.Service)))
```

Comment:
I noticed that. The PR is just very initial. It's just detecting OpenRC, it's not even guaranteed to work propertly. Becuase kubeadm still writes a service file. I need to modify this to write a conf.d file for OpenRC. There is still more work ...
Since you have no objection, I'd leave this, and I will send more patches soon to get kubelet running with OpenRC.